pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -318,6 +318,18 @@ type CommonPrometheusFields struct {
 	// It requires Prometheus >= v2.33.0.
 	EnableRemoteWriteReceiver bool `json:"enableRemoteWriteReceiver,omitempty"`
 
+	// Enable Prometheus to be used as a receiver for the OTLP Metrics protocol.
+	//
+	// WARNING: This is not considered an efficient way of ingesting samples.
+	// Use it with caution for specific low-volume use cases.
+	// It is not suitable for replacing the ingestion via scraping and turning
+	// Prometheus into a push-based metrics collection system.
+	// For more information see https://prometheus.io/docs/prometheus/3.0/querying/api/#otlp-receiver
```

Comment:
I would remove the warning notice since Prometheus 3 supports this officially.
```suggestion
```

Reply:
Thanks. Remove it done.